ovc5/scripts/install_sd.sh

```diff
@@ -157,9 +182,7 @@ mount_drive () {
 }
 
 copy_bin () {
-  sudo cp $BOOT_FILES_DIR/BOOT.BIN $BOOT_DIR/boot.bin
-  sudo cp $BOOT_FILES_DIR/image.ub $BOOT_DIR
-  sudo cp $BOOT_FILES_DIR/boot.scr $BOOT_DIR
+  sudo cp $SD_BOOT_FILES_DIR/* $BOOT_DIR/
```

Comment:
This is the only line I haven't tested but I'm going to assume it works. @luca-della-vedova feel free to give it all a run-through!

Reply:
Is this still not tested? It looks about right to me, my testing setup is a bit messed up right now since I'm transitioning the Vivado design from the devkit to the OVC5 PCB, hence everything is pretty much broken.